project/Dependencies.scala

```diff
@@ -88,6 +88,8 @@ object Dependencies {
     "org.mongodb" %% "casbah" % "3.0.0"
   )
 
+  val sparkBackendDependencies = List("org.json4s" %% "json4s-jackson" % "3.4.0")
```

Comment:
I don't think "it's easier to use" use is a good argument for doing something completely differently from the rest of the code base